city_tweet_mood.js

```diff
@@ -1,92 +1,96 @@
-'use strict';
-const path = require('path');
-const process = require('process');
-process.env.GOOGLE_APPLICATION_CREDENTIALS = process.cwd() + path.sep + 'google_sa.json';
-const twitter_secrets = require('./twitter_keys');
-const language = require('@google-cloud/language');
-const Twitter = require('twitter');
+"use strict";
+const path = require("path");
+const process = require("process");
+process.env.GOOGLE_APPLICATION_CREDENTIALS =
+  process.cwd() + path.sep + "google_sa.json";
+const twitter_secrets = require("./twitter_keys");
+const language = require("@google-cloud/language");
+const Twitter = require("twitter");
 const google_client = new language.LanguageServiceClient();
 const twitter_client = new Twitter({
-    consumer_key: twitter_secrets.TWITTER_API,
-    consumer_secret: twitter_secrets.TWITTER_SECRET,
-    access_token_key: twitter_secrets.TWITTER_ACCESS_TOKEN,
-    access_token_secret: twitter_secrets.TWITTER_ACCESS_SECRET
+  consumer_key: twitter_secrets.TWITTER_API,
+  consumer_secret: twitter_secrets.TWITTER_SECRET,
+  access_token_key: twitter_secrets.TWITTER_ACCESS_TOKEN,
+  access_token_secret: twitter_secrets.TWITTER_ACCESS_SECRET
 });
-
+const cities = require("./cities");
 
 /**
- * TODO - make a city -> coordinate database
- * @param city_name
+ * @param city_name string - city to analyze
+ * @param state_name string - state of city to analyze
  * @returns string of gps coordinates (lat,long)
  */
-function get_city_coordinates(city_name) {
-    // PLACEHOLDER FOR GETTING THE CITY COORDINATES VIA API
-    return '45.523,-122.676';
+function get_city_coordinates(city_name, state_name) {
+  let city = cities.find(
+    data => data.city.match(city_name) && data.state.match(state_name)
+  );
+  var lat = city.latitude.toString();
+  var lng = city.longitude.toString();
+  return lat + "," + lng;
```

Comment:
nitpicky: need to be vars? could probably just do `return city.latitude.toString() + ',' + city.longitude.toString()`